installer/scripts/common/setupIstio.sh

```diff
@@ -1,23 +1,39 @@
 #!/bin/bash
 source ./common/utils.sh
 
-kubectl create namespace istio-system
+kubectl get ns istio-system
```

Comment:
Is istio **always**  installed into `istio-system`, or is there a way to determine if istio is installed into another namespace?

Reply:
Istio can be installed in any namespace. However, in this first version, we only check for the `istio-system` namespace.